Platforms/QemuQ35Pkg/CpuInfoDxe/CpuInfoDxe.inf

```diff
@@ -0,0 +1,35 @@
+## @file CpuInfoDxe.inf
+#
+# Cpu Info Debug Dxe
+#
+#Copyright (c) Microsoft Corportaion. All rights reserved.
+#
+##
+
+[Defines]
+   INF_VERSION       = 1.27
+   BASE_NAME         = CpuInfoDxe
+   FILE_GUID         = F5E4F8D3-9975-4263-8BB5-67178988429E
+   VERSION_STRING    = 1.0
+   MODULE_TYPE       = DXE_DRIVER
+   ENTRY_POINT       = CpuInfoDxeEntryPoint
+
+[Sources]
+  CpuInfoDxe.c 
+
+[Packages]
+  MdePkg/MdePkg.dec
+  QemuQ35Pkg/QemuQ35Pkg.dec
+  MdeModulePkg/MdeModulePkg.dec
+  UefiCpuPkg/UefiCpuPkg.dec

```

Comment:
Arrange in descending order from generic to specific so potential conflicts resolve as expected.

```suggestion
  MdePkg/MdePkg.dec
  MdeModulePkg/MdeModulePkg.dec
  UefiCpuPkg/UefiCpuPkg.dec
  QemuQ35Pkg/QemuQ35Pkg.dec

```